_posts/en/posts/2025-04-14-release-29.0.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Bitcoin Core 29.0 released
+name: blog-release-29.0
+id: en-blog-release-29.0
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 29.0 is now available.
+---
+Bitcoin Core version 29.0 is now available for [download][download
+page].  See the [release notes][release notes] for more information about the
+bug fixes in this release.
+
+With the release of this new major version, versions 26.x and older are at
+"Maintenance End" and will no longer receive updates or security fixes. In
```

Comment:
nit: this might imply that we normally backport security fixes to non-EOL older versions, but we don't.

Reply:
are you suggesting delete "or security fixes" ?

Reply:
Yes.

Reply:
removed